mz_bokeh_package/components/plot_settings.py

```diff
@@ -704,6 +713,24 @@ def _init_plot_settings_values(self):
             self._set_setting_property(setting_id, value)
             self._set_setting_widget_value(setting_id, value)
 
+    def _set_color_for_renderer(self, renderer_name, new_color):
+        """Set line color and fill color to a renderer default presentaion, on selction,
+        and not selected plyphs.
```

Comment:
```suggestion
        """Set line- and fill color of a bokeh renderer for its default representation, as well as for the selection
        and nonselection glyphs.
```
(Maybe check that only the GlyphRenderer has a selection and nonselection glyph and if we can expect any other renderer here. And then specifiy that here in the docstring, the type hint for the parameter and also in the code (`if renderer in renderers_with_selection_glyphs:...`) if necessary.)

Reply:
docstring edited and a check that the renderer has the selection properties was added.